source/solvers/vof.cc

```diff
@@ -1792,23 +1767,24 @@ VolumeOfFluid<dim>::assemble_L2_projection_interface_sharpening(
                   // $$ (if c <  \phi <= 1)  {\Phi = 1 - (1 - c) ^ (1 -
                   // \alpha)
                   // * (1 - \phi) ^ \alpha}
-                  if (phase_value >= 0.0 && phase_value <= sharpening_threshold)
+                  if (phase_value >= 0 and phase_value <= sharpening_threshold)
```

Comment:
why change it? I know they are synonyms but I gathered that `&&` is more conventional and `and` is used when there is a problem with using & symbol

Reply:
I find and more verbose, but I can revert it back.

Reply:
no it's ok, but we should ideally be consistent throughout the code about it